encoder-otel-brave/src/main/java/zipkin2/reporter/otel/brave/OtelEncoder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package zipkin2.reporter.otel.brave;
+
+import brave.Tag;
+import brave.handler.MutableSpan;
+import io.opentelemetry.proto.trace.v1.TracesData;
+import zipkin2.reporter.BytesEncoder;
+import zipkin2.reporter.Encoding;
+
+@SuppressWarnings("ImmutableEnumChecker") // because span is immutable
+public class OtelEncoder implements BytesEncoder<MutableSpan> {
```

Comment:
Not sure but an idea, maybe instead of `Otel`, `Otlp` such as `OtlpProtoV1Encoder`?

Also wonder about leaving out v1 for now while still adding v2 in the future if needed. There's no v1 in any specs or sdk implementations in OTel so I feel it could cause some confusion.

Reply:
Actually I guess there is a v1 in the package name so never mind on that 😅

Reply:
OtlpProtoV1Encoder works great for me. I think otlp is the more clarifying thing.

Reply:
Renamed to `OtlpProtoV1Encoder` in https://github.com/openzipkin-contrib/zipkin-otel/pull/9/commits/6301d1cbd897bc7e98b665b3993947317b37b4bb